src/homework/day8/Chairs.java

```diff
@@ -0,0 +1,56 @@
+
+package homework.day8;
+
+import homework.day8.classes.Bubble;
+import homework.day8.classes.Chair;
+import homework.day8.classes.Sand;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Chairs {
+    public static void main(String[] args) {
+        List<Chair> chairs = Arrays.asList(
```

Comment:
Could you please rename list to "furniture" as per requirement? (But not sure that this is 100% crucial)